src/services/Twitter.js

```diff
@@ -0,0 +1,20 @@
+const Twit = require('promised-twit')
+const getBase64 = require('./getBase64Url')
+const T = new Twit({
+  consumer_key: process.env.TWITTER_API_KEY,
+  consumer_secret: process.env.TWITTER_SECRET_KEY,
+  access_token: process.env.TWITTER_ACCESS_TOKEN,
+  access_token_secret: process.env.TWITTER_ACCESS_TOKEN_SECRET
+})
+
+module.exports.postTweet = async (politician) => {
+  // first we must post the media to Twitter
+  const b64content = await getBase64(politician.profilePicture)
+  const data = await T.postMediaUpload({ media_data: b64content })
+  const status = `
+    ${politician.name} (${politician.twitter}) está na lista de políticos que apoiam medidas anti-ciência. ${process.env.API_DOMAIN}/detalhes/${politician.slug} #ListaSujaDoCorona
```

Comment:
How about checking whether we have a twitter account saved for this politician before? I'd say check it before, because we may have politicians without a twitter account in the future